devicetypes/smartthings/zwave-smoke-alarm.src/zwave-smoke-alarm.groovy

```diff
@@ -146,6 +146,13 @@ def zwaveEvent(physicalgraph.zwave.commands.alarmv2.AlarmReport cmd, results) {
 			if(device.currentValue("smoke") != "clear") {
 				createSmokeOrCOEvents("smokeClear", results)
 			}
+
+			if (device.currentValue("carbonMonoxide") != "clear") {
+				def zwMap = getZwaveInfo()
+				if (zwMap && zwMap.mfr == "0138" && zwMap.prod == "0001" && zwMap.model == "0002") {
+					createSmokeOrCOEvents("carbonMonoxideClear", results)
+				}
```

Comment:
I also just updated the code to explicitly solicit the alarm states, since we should be doing this regardless if the alarm gives them to us (as other alarms may not).